CONTRIBUTING.md

```diff
@@ -85,55 +83,22 @@ query = transformer.transform(tree)
 print(query)
 ```
 
-```python
-{'$and': [{'_mp_bandgap': {'$gt': 5.0}}, {'_cod_molecular_weight': {'$lt': 350.0}}]}
+```json
+{
+    '$and': [
+        {'_mp_bandgap': {'$gt': 5.0}},
+        {'_cod_molecular_weight': {'$lt': 350.0}}
```

Comment:
```suggestion
    "$and": [
        {"_mp_bandgap": {"$gt": 5.0}},
        {"_cod_molecular_weight": {"$lt": 350.0}}
```

Reply:
Was about to say well-spotted, then I realised the diff made this quite an obvious error :grin:

Reply:
Hehehe, indeed. The work was already done for me here 😅